src/Host/Client/Test/DataInspection/ValuesTest.cs

```diff
@@ -31,13 +31,14 @@ public class ValuesTest : IAsyncLifetime {
         private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
 
         private readonly MethodInfo _testMethod;
+        private readonly IRHostBrokerConnector _brokerConnector = new RHostBrokerConnector(name: nameof(ValuesTest));
```

Comment:
> Order is actually explicit for instance fields as well

Yes, compiler defines it explicitly, but in case of derived classes I have to think about it. If we have `A : B` and `B : C`, it will first initialize fields for `A`, then for `B`, then for `C`, then it will call constructor for `C`, then for `B`, then for `A`. I just want the order to be simpler :)

---
In reply to: [75028449](https://github.com/Microsoft/RTVS/pull/2173#discussion_r75028449) [](ancestors = 75028449,75027791,75011847,74972244)